hack/common.sh

```diff
@@ -711,11 +711,11 @@ os::build::ldflags() {
 
   GO_VERSION=($(go version))
   if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.5') ]]; then
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit" "${KUBE_GIT_COMMIT}")
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion" "${KUBE_GIT_VERSION}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit" "${OS_GIT_COMMIT}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion" "${OS_GIT_VERSION}")
   else
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit=${KUBE_GIT_COMMIT}")
-    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion=${KUBE_GIT_VERSION}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitCommit=${OS_GIT_COMMIT}")
+    ldflags+=(-X "k8s.io/kubernetes/pkg/version.gitVersion=${OS_GIT_VERSION}")
```

Comment:
> I vaguely liked having the info available, but I'll let @smarterclayton weigh in

Do you have any good advice for making sure that we never accidentally start using after rebases?  Using it in places like command line clients (but only for some resources), `MatchesServerVersion` (but only sometimes), `/version` endpoint?  Mechanics-wise it is painful to keep and of dubious value.  When have you used it?  Was it more useful than knowing the origin level?